utils/create_transactions.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+#
+# Create cucumber example tables for app transactions
+#
+# Must have an applications-enabled version of goal on the path, and any running node.
+
+
+# this is mandatory, 'goal app' currently only supports looking up fee/first-valid/last-valid
+DATA_DIR="/home/will/algorand/networks/applications-test/Node"
```

Comment:
will need to be updated before merge, right?

Reply:
This is just used as a utility to generate the feature file table, so it's mostly for reference in case something similar is useful in the future.

Reply:
This will be of great value to have for reuse, reference, or learning. 
Please do one more step of removing local reference (e.g. /home/will/...) and make it parametrized.